.github/workflows/risc0.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       uses: actions/cache@v3
       with:
         path: ~/.cache/zig
-        key: ${{ runner.os }}-zig-packages-${{ hashFiles('build.zig.zon') }}
+        key: ${{ runner.os }}-zig-packages-${{ hashFiles('**/build.zig.zon') }}
```

Comment:
That might be a useful fix, but I'd like to have it in another PR to speed things up.